### DISCUSSION TITLE
Haven't still received the benefits of the student development pack

### DISCUSSION THREAD

**Discussion by Fcatilizer** [community/community#138624] [GitHub Education]
### Hi everyone,

It took 4 attempts to get approved due to 2FA not enabled. After 2FA setup, I re-uploaded the student Id and it got approved. It's been 3 days but I haven't got access of features like Github Copilot.
![git](https://github.com/user-attachments/assets/b43d72a7-f73a-4808-bb0f-2c499d8d4b9b)

**Comment by davevad93** [comment]
Hi @Fcatilizer ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Fcatilizer was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by AyushKumarVujani** [comment]
where can i apply student development pack ? im new

**Reply by davevad93** [reply]
Hey @AyushKumarVujani , you can apply [here](https://education.github.com/discount_requests/application), also take a look at the requirements to [apply as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student). 

If you encounter any issues during the application process, check out the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352), where you'll find a lot of useful information.

**Comment by pg56714** [comment]
@Fcatilizer 
Mark me as answer thanks

[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)
